app/views/application/_nav_right.html.slim

```diff
@@ -15,6 +15,7 @@ ul.nav.navbar-nav.navbar-right
       - if current_season.started?
         li.divider.hidden-xs
         li class=active?(:conferences) = link_to 'Conferences',  conferences_path
+        li class=active?(:projects) = link_to 'Projects', projects_path
```

Comment:
I just realized that the projects link is also missing during the Season.projects_proposable phase (Line 8) : can you also add the link to that menu? You can put it under the  link to the new_project_path.

If you feel up to adding specs for that too, that would be great, but I leave that up to you. Either way is cool!